python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -402,8 +402,14 @@ def do_evaluate(
         mapping: Mapping[Expr, Column] | None = None,
     ) -> Column:
         """Evaluate this expression given a dataframe for context."""
-        # TODO: type is wrong, and dtype
-        return df.num_rows  # type: ignore
+        return Column(
+            plc.Column.from_scalar(
+                plc.interop.from_arrow(
```

Comment:
I see this twice, if you anticipate it coming up again maybe worth a short helper function? Not sure.